package.json

```diff
@@ -38,14 +38,14 @@
     "@ionic/prettier-config": "^2.0.0",
     "commitizen": "^4.2.4",
     "cz-conventional-changelog": "^3.3.0",
-    "eslint": "^7.32.0",
+    "eslint": "^8.57.0",
     "eslint-config-prettier": "^8.3.0",
     "eslint-plugin-import": "^2.25.3",
     "eslint-plugin-prettier": "^4.0.0",
     "husky": "^7.0.4",
-    "lerna": "^4.0.0",
+    "lerna": "^8.1.8",
```

Comment:
Can you explain why Lerna 8 is required to support Angular 18? I would like to avoid this change since `lerna bootstrap` was removed in v7 and this change will break CI based on current changes.

Reply:
It is not required, but Lerna 4 is no longer supported. 

For me the CI part is unclear and it seems unavailable to me as I can only contribute using a PR.

I will revert back to vs 4.0.0.